src/components/BreedsList.vue

```diff
@@ -256,39 +255,40 @@ export default {
   },
   watch: {
     $route(newRoute, oldRoute) {
+      console.log("watch route");
       if (this.$route.name === "breedName") {
         this.$router.push({
           name: "details",
         });
       }
+      console.log("new", newRoute, "old", oldRoute);
+      console.log(newRoute);
+
+      this.lastBreedState = [];
+      this.indexClicked = 0;
+      this.displayedBreeds.forEach((breed) => {
+        breed.isActive = false;
+      });
+    },
+
+    displayedBreeds: {
+      handler(breeds) {
+        breeds.filter((breed, id) => {
+          if (breed.isActive) {
+            this.$refs.resultListContainer.children[id].scrollIntoView({
```

Comment:
watchat computed property je mal tko tko.. če se le da, bi blo boljš rešt drugač :D 

fora je ker computed že sam po sebi "watcha" dependancies k jih uporablja in se re-runa ko se spremenijo ane.. pol pa da še to watchaš.. velik se dogaja, velikokrat po nepotrebnem..

ne pravim pa da je to v tem primeru, mal razmisli kako bi mogoče lahko še drugače rešu in če najdeš rešitev, super

Reply:
Maš prov, bom razmišlu, kako bi drugače rešu zadevo